scripts/jenkins/gitlint.ini

```diff
@@ -8,13 +8,10 @@ line-length=80
 words=wip,WIP
 
 [title-match-regex]
-regex=^(?!SOC).*\(((bsc#|SOC-)[0-9]+(,\s)?)+\)$
+regex=^(?!SOC).*\((((bsc#|SOC-)[0-9]+(,\s)?)+|trivial)\)$
```

Comment:
is there corresponding filtering on "(trivial)" in the changelog extraction code? because I think we shouldn't have this in customer visible changelogs..

Reply:
Why not? We could use a different word if this is what you want to say.